core/src/avm1/globals/xml.rs

```diff
@@ -1066,6 +1074,51 @@ pub fn xml_status<'gc>(
     Ok(Value::Undefined)
 }
 
+fn spawn_xml_fetch<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    this: Object<'gc>,
+    loader_object: Object<'gc>,
+    url: &AvmString,
+    send_object: Option<XmlNode<'gc>>,
+) -> Result<Value<'gc>, Error<'gc>> {
+    let request_options = if let Some(node) = send_object {
+        // Send `node` as string
+        RequestOptions::post(Some((
+            node.into_string(&mut is_as2_compatible)
+                .unwrap_or_default()
+                .into_bytes(),
+            "application/x-www-form-urlencoded".to_string(),
+        )))
+    } else {
+        // Not sending any parameters.
+        RequestOptions::get()
+    };
+
+    this.set("loaded", false.into(), activation)?;
+
+    let fetch = activation.context.navigator.fetch(&url, request_options);
+    let target_clip = activation.target_clip_or_root()?;
+    // given any defined loader object, sends the request. Will load into LoadVars if given.
+    let process = if let Some(node) = loader_object.as_xml_node() {
+        activation.context.load_manager.load_xml_into_node(
+            activation.context.player.clone().unwrap(),
+            node,
+            target_clip,
+            fetch,
+        )
+    } else {
+        activation.context.load_manager.load_form_into_load_vars(
+            activation.context.player.clone().unwrap(),
+            loader_object,
+            fetch,
+        )
+    };
```

Comment:
Is this part left over from LoadVars? I think this should be unreachable, so perhaps the `spawn_xml_fetch` should take a `xml_result: XmlNode` directly.

Reply:
Ah, I see, you can actually pass a `LoadVars` along here and it receives the data just the same.